### ISSUE TITLE
AVT 3 - Tag w/ Filter React Component VO on macOS and iOS the Clear Filter button is not announced correctly.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4427]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon React Version 7.7.1

## Description
1. Control-Option-Shift-Down Arrow to enter the Web area
2. Control-Option Right Arrow. VO announces the label and that it is a text element.
3. Tab to the "X". VO does not say anything.
4. Press Command-R to reload the page
5. Press Tab to the "X", VO announces, "Clear Filter, you are currently on a group. To interact with items in this group press Control-Option-Shift-Down Arrow."
<img width="731" alt="Screen Shot 2019-10-22 at 2 15 40 PM" src="https://user-images.githubusercontent.com/21676914/67325408-ba783a00-f4da-11e9-8db9-1835478ad31a.png">

**Expected Result:**
After tabbing to the "X", VO should announce, "Clear Filter, button"

## Environment
iOS Version 13.1.3 with Safari and VO
Carbon React Version 7.7.1

## Description
Swipe Right to the "X". VO announces "Clear Filter Image."
**Expected Result:** 
AfterSwiping Right to the "X", VO should announce, "Clear Filter, button"
